test_login.py

```diff
@@ -665,24 +665,21 @@ def test_remember_me_uses_custom_cookie_parameters(self):
             self.assertLess(difference, timedelta(seconds=10), fail_msg)
             self.assertGreater(difference, timedelta(seconds=-10), fail_msg)
 
-    def test_remember_me_with_invalid_duration_returns_500_response(self):
```

Comment:
We still need this test, but instead of setting to an integer it should set to something other than int or timedelta.